### ISSUE TITLE
[Feature Request] Qualify Accounts by max followers

### ISSUE THREAD

**Issue by TheNeObr** [InstaPy/InstaPy#4627]
## Feature Creation

Feature - Qualify Accounts by max followers and log 

Function to pick business/Influencer account and make a log from that, like username and followers ?

i will like to pick on a selected region all business account with x followers like eg. 5000+

Document this and use to make my strategy use follow by user.

so it would be easier to reach the target audience by targeting the large accounts that exist in the target region, making it easier to create a strategy to more accurately target the avatar that I'm looking for.

Basically doing the same as the Ads but in a smarter way, and at no cost.

I apologize if the writing is strange I use google to write because I am not native to the language, I believe it is understandable.

[ ]´s
NeO
